tests/validator/validator.cpp

```diff
@@ -1200,6 +1278,40 @@ TEST(Validator, resetNoTestValue)
     EXPECT_EQ_ERRORS(expectedErrors, validator);
 }
 
+TEST(Validator, resetMultipleMathMlInTestValue)
```

Comment:
Casing, it should be: `resetMultipleMathMLInTestValue `.

Reply:
Gah ... in earlier PRs I was told it went the other way  ... :|

Reply:
TBH, I would personally use `resetMultipleMathmlInTestValue`, but I have seen `xxxYyyyMathMLZzzz` in quite a few places.

Reply:
I've changed it once ... and you have edit permission ... so you can feel free to change it again if the mood takes you ;)